site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -500,7 +500,7 @@
       "source": [
         "## Manually save weights\n",
         "\n",
-        "Manually saving weights with the `Model.save_weights` method. By default, `tf.keras`—and `save_weights` in particular—uses the TensorFlow [checkpoint](../../guide/checkpoint.ipynb) format with a `.ckpt` extension (saving in [HDF5](https://js.tensorflow.org/tutorials/import-keras.html) with a `.h5` extension is covered in the [Save and serialize models](https://www.tensorflow.org/guide/keras/save_and_serialize#weights-only_saving_in_savedmodel_format) guide):"
+        "Manually saving weights with the `Model.save_weights` method. By default, `tf.keras`—and `save_weights` in particular—uses the TensorFlow [checkpoint](../../guide/checkpoint.ipynb) format with a `.ckpt` extension (saving in [HDF5](https://www.tensorflow.org/guide/keras/save_and_serialize#hdf5_format) with a `.h5` extension is covered in the [Save and serialize models](https://www.tensorflow.org/guide/keras/save_and_serialize#weights-only_saving_in_savedmodel_format) guide):"
```

Comment:
Thanks!

A few fixes:

- Just to make sure the link works in case there's a change in the section title (like https://www.tensorflow.org/guide/keras/save_and_serialize#weights-only_saving_in_savedmodel_format which no longer takes you to the right section).
- Since the `serialization_and_saving` guide is hosted on https://github.com/keras-team/keras-io/blob/master/guides/serialization_and_saving.py, we'll use a full link instead of a relative link. (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks) 

```suggestion
        "To save weights manually, use `tf.keras.Model.save_weights`. By default, `tf.keras`—and the `Model.save_weights` method in particular—uses the TensorFlow [Checkpoint](../../guide/checkpoint.ipynb) format with a `.ckpt` extension. (To save in the HDF5 format with a `.h5` extension, refer to the [Save and load models](https://www.tensorflow.org/guide/keras/save_and_serialize) guide):"
```